### PR TITLE
[DEV-175] Give API providers redirect url via API change

### PR DESCRIPTION
**Pull request by bimonXdd**

### CODE REVIEW

**Comment by Nuffic** [comment]
I would still like to see a point in the "Lead registration flow" that the customer has to be redirected to us as the last step. This is to avoid any confusion with it.

**Comment by Nuffic** [comment]
Philippines affiliates' is missing

File: affiliates/argentina.swagger.json [hunk 1]
```diff
@@ -451,6 +451,10 @@
             "product" : {
               "type" : "string",
               "example" : "PAYDAY"
+            },
+            "provider_redirect_url" : {
```
Comment (mikk150) [inline affiliates/argentina.swagger.json]: this is not how it works, we have `_links` object between it

**Review by mikk150 — CHANGES_REQUESTED** [review]

File: affiliates/argentina.swagger.json (hunk 1 above)
Comment (bimonXdd) [reply affiliates/argentina.swagger.json]: Added _links object in between

**Review by Nuffic — CHANGES_REQUESTED** [review]
image

Gives error when parsing. 
Also I would not leave the same description for the redirection point in API flow. The url is now dynamic and has to be taken from _links property instead.

**Review by mikk150 — APPROVED** [review]

**Review by Nuffic — APPROVED** [review]

**Pull request merged**